app/main.py

```diff
@@ -2,0 +1,55 @@
+class SoftwareEngineer:
+    skills = []
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    skills = ["JavaScript", "HTML", "CSS"]
```

Comment:
Similar to the `SoftwareEngineer` class, `skills` here is a class variable. Consider making it an instance variable to avoid shared state across instances.